platform/jobs/pipelines/edxPlatformPipelineMasterWTWNightly.groovy

```diff
@@ -33,9 +33,13 @@ jobConfigs.each { jobConfig ->
                 PYTHON_VERSION: "${jobConfig.pythonVersion}"
             )
 
-            triggers {
-                cron('@daily')
-            }
+            // Turn off job until we're ready to work on it again; at
```

Comment:
1. "We should either delete this check or exclude it from the status checks, but for now I'm just disabling the daily run on master so that we can defer that decision." feels like useful context to add here or to a ticket.
2. I'm not clear on the downside of deleting this altogether. If we want it - I feel like we should ticket it and reference the ticket from the comment. The ticket can include additional comments/thoughts about fixing or permanently deleting.

Reply:
Downside is messing up git blame history. My plan here is to disable for now to get it out of the way and then notify the owner; they can either delete or fix.

Reply:
I was assuming we were the owner because we own edx-platform.  I don't actually understand what the "who-tests-what" check is. :)

Reply:
I think Jeremy said that Ned would be the person to talk to, and that Cale had been leading this.

Reply:
WTW is a project to collect coverage information from tests so that we know which parts of the codebase are covered by which tests. I think. Something like that.

Reply:
WTW would collect data about which tests ran each line of product code.  It's expensive, and never got to a working state.  I don't know who might pick it up next, but it probably will not be for a long time, if ever.  I would support deleting this.  There's also a dependency in edx-platform that we could remove:
```
https://github.com/nedbat/coverage_pytest_plugin.git
```

Reply:
Is there a Confluence page or anything where we could point to the clean-up PRs?

Reply:
I'm not finding any relevant wiki pages.